### PR TITLE
[MINI-5418] Maximum storage size limit should be set in Bytes instead of Mega Bytes

### PR DESCRIPTION
**Pull request by gopi-karmakar**
# Description
Describe the changes in this pull request.
Explain your rationale of technical decisions you made (unless discussed before).

## Links
Add links to github/jira issues, design documents and other relevant resources (e.g. previous discussions, platform/tool documentation etc.)

# Checklist
- [ ] I have read the [contributing guidelines](https://github.com/rakutentech/android-miniapp/blob/master/.github/CONTRIBUTING.md).
- [ ] I have completed the [SDK Development Learning Path](https://github.com/rakutentech/android-miniapp/blob/master/.github/CONTRIBUTING.md#sdk-development-learning-path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [ ] I removed all sensitive data before every commit, including API endpoints and keys
- [ ] I ran `./gradlew check` without errors

### CODE REVIEW

**Review by munir-rakuten — CHANGES_REQUESTED** [review]
Please update the change log and userguide as well.

**Review by laubengaier — APPROVED** [review]
lgtm, I'll approve but changelog update is necessary as munir mentioned

**Comment by gopi-karmakar** [comment]
> lgtm, I'll approve but changelog update is necessary as munir mentioned

@laubengaier Change log and user guide has been updated. Kindly check and if looks fine then Kindly approve.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,9 @@
 - **Feature:** Added `MiniAppView` to create multiple MiniApp views.
 - **Feature:** Added `MiniAppConfig` data class to hold configuration settings for the `MiniAppView`.
 - **Deprecated:** Old `create` and `createWithUrl` interfaces.
+- **Fix:** Made the `MaxStorageSize` Limit in Bytes.
+- **Fix:** Returning `onSuccess` for the Bulk Delete even if no items could be deleted.
+- **Update:** Refactored the variable name to `maxStorageSizeLimitInBytes` and `newMiniAppSdkConfig`
```
Comment (munir-rakuten) [inline CHANGELOG.md:10]: @gopi-karmakar the last one will be in sample app changelog.

Comment (gopi-karmakar) [reply CHANGELOG.md:10]: Actually, the `maxStorageSizeLimitInBytes` is not the Sample app parameter but it's the `MiniAppSdkConfig` parameter so it's important I think. 
Yes, I agree `newMiniAppSdkConfig`  related to the sample app I can remove this one.

Comment (munir-rakuten) [reply CHANGELOG.md:10]: yeah right, i have changed my comments, please remove it .

Comment (gopi-karmakar) [reply CHANGELOG.md]: @munir-rakuten Updated, Kindly check.

**Review by munir-rakuten — APPROVED** [review]

**Pull request merged**